docs/core/tutorial/04-handling-failure.md

```diff
@@ -11,34 +11,34 @@ python 04_handle_failures.py
 
 ## If at first you don't succeed...
 
-Now that we have a working ETL Flow let's take further steps to ensure it's robustness. The `extract_*` Tasks are making web requests to external API's in order to fetch the data. What if the API is unavailable for a short period? Or if a single request times out for unknown reasons? **Prefect `Tasks` can be retried on failure**; let's add this to our `extract_*` tasks:
+Now that we have a working ETL Flow let's take further steps to ensure its robustness. The `extract_*` Tasks are making web requests to external APIs in order to fetch the data. What if the API is unavailable for a short period? Or if a single request times out for unknown reasons? **Prefect `Tasks` can be retried on failure**; let's add this to our `extract_*` tasks:
 
 ```python{1,6,12}
 from datetime import timedelta
 import aircraftlib as aclib
 from prefect import task, Flow, Parameter
 
 
-@task(max_retries=3, retry_delay=timedelta(seconds=10))
+@task(max_retries=3, retry_delay=timedelta(seconds=1))
```

Comment:
Ideally we should leave this at 10 and change the code example to 10 seconds to match, but good catch regarding consistency

Reply:
done in e60967fe